types/aframe/tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
The tests have a dependency on the intern test framework. Do you have a suggestion on how I can provide that dependency so people can run the tests?